eslint/eslint-plugin/src/no-untyped-underscore.ts

```diff
@@ -43,7 +43,7 @@ const noUntypedUnderscoreRule: TSESLint.RuleModule<MessageIds, Options> = {
         // Is it an expression like "x.y"?
 
         // Ignore expressions such as "super.y", "this.y", and "that.y"
-        const memberObject: TSESTree.LeftHandSideExpression = node.object;
+        const memberObject: TSESTree.Expression = node.object;
```

Comment:
Is there a unit test to confirm that this rule is still working correctly?

Reply:
There is now.